src/app/dev/DevToys.Tools/Tools/Text/ListCompare/ListCompareGuiTool.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System.Security.Authentication;
+using System.Security.Cryptography;
+using DevToys.Api;
+using DevToys.Tools.Helpers;
+using DevToys.Tools.Models;
+using DevToys.Tools.Tools.EncodersDecoders.Base64Text;
+using DevToys.Tools.Tools.Generators.HashAndChecksum;
+using Microsoft.Extensions.Logging;
+
+namespace DevToys.Tools.Tools.Text.ListCompare;
+
+[Export(typeof(IGuiTool))]
+[Name("ListCompare")]
+[ToolDisplayInformation(
+    IconFontName = "DevToys-Tools-Icons",
+    IconGlyph = '\u0137',
+    GroupName = PredefinedCommonToolGroupNames.Text,
+    ResourceManagerAssemblyIdentifier = nameof(DevToysToolsResourceManagerAssemblyIdentifier),
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Text.ListCompare.ListCompare",
+    ShortDisplayTitleResourceName = nameof(ListCompare.ShortDisplayTitle),
+    LongDisplayTitleResourceName = nameof(ListCompare.LongDisplayTitle),
+    DescriptionResourceName = nameof(ListCompare.Description),
+    AccessibleNameResourceName = nameof(ListCompare.AccessibleName))]
+internal sealed class ListCompareGuiTool : IGuiTool
+{
+    /// <summary>
+    /// Whether the list comparison should be case sensitive or not,
+    /// </summary>
+    private static readonly SettingDefinition<bool> caseSensitive
+        = new(
+            name: $"{nameof(ListCompareGuiTool)}.{nameof(caseSensitive)}",
+            defaultValue: false);
+
+    private CancellationTokenSource? _cancellationTokenSource;
```

Comment:
Please move this line down to group it with other fields.

Reply:
Sorry, I was meaning to move `_cancellationTokenSource` down to line 42, to group it with fields _listA and _listB.
Let's also keep setting definition groups. You can move `comparisonMode` back up here.